modules/zinit/service.go

```diff
@@ -3,16 +3,40 @@ package zinit
 import (
 	"fmt"
 	"io/ioutil"
+	"os"
 	"path/filepath"
+	"strings"
 
 	"gopkg.in/yaml.v2"
 )
 
+// LogType is an enum type
+type LogType string
+
+// UnmarshalYAML implements the  yaml.Unmarshaler interface
+func (s *LogType) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	var buf string
+	if err := unmarshal(&buf); err != nil {
+		return err
+	}
+	*s = LogType(strings.ToLower(buf))
+	return nil
+}
+
+// All the type of logging supported by zinit
+const (
+	NullLogType   LogType = "null"
```

Comment:
We don't have the null type anymore after your request to drop it :D

Reply:
ha ok, Then we need to udpate the documentation at: https://github.com/threefoldtech/zinit/tree/master/docs cause the null mode is still present there